ds248x/ds248x.go

```diff
@@ -154,6 +161,70 @@ func (d *Dev) Search(alarmOnly bool) ([]onewire.Address, error) {
 	return onewire.Search(d, alarmOnly)
 }
 
+// ChannelSelect function is for selecting one of eight 1-w channels on DS2482-800. 
+// On other chips it does nothing. Function silently limits channel selection between
+// 0 and 7. It is expected that application keeps track of
+// with 1-w device is connected to with channel.
+//
+// Communication error is returned if present.
+//
+func (d *Dev) ChannelSelect(ch int) (err error) {
+	switch d.isDS248x {
+	case isDS2482x100:
+
```

Comment:
return nil if ch == 0, otherwise an error? Same for isDS2483.

Reply:
nil is returned by default. I'm a bit reluctant to produce too many errors. What wrong with doing nothing on these particular chips?

Reply:
Silent failure is super hard to diagnose, and it's even harder if the user is not well versed in Go.